src/main/java/com/sweepy/urlService/urlService.java

```diff
@@ -1,7 +1,6 @@
 package com.sweepy.urlService;
 
 import javax.servlet.http.HttpServletResponse;
-import javax.swing.*;
 import java.io.IOException;
 
 public interface urlService {
```

Comment:
The class name should always start with capital letter, only function/variable starts with lower case. Please change for all the ones applicable

Reply:
addressed